sdk/iot/Azure.ResourceManager.IotFirmwareDefense/api/Azure.ResourceManager.IotFirmwareDefense.net8.0.cs

```diff
@@ -3,22 +3,22 @@ namespace Azure.ResourceManager.IotFirmwareDefense
     public partial class FirmwareAnalysisSummaryCollection : Azure.ResourceManager.ArmCollection, System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.IotFirmwareDefense.FirmwareAnalysisSummaryResource>, System.Collections.Generic.IEnumerable<Azure.ResourceManager.IotFirmwareDefense.FirmwareAnalysisSummaryResource>, System.Collections.IEnumerable
     {
         protected FirmwareAnalysisSummaryCollection() { }
-        public virtual Azure.Response<bool> Exists(Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareAnalysisSummaryName summaryName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareAnalysisSummaryName summaryName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Response<Azure.ResourceManager.IotFirmwareDefense.FirmwareAnalysisSummaryResource> Get(Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareAnalysisSummaryName summaryName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<bool> Exists(Azure.ResourceManager.IotFirmwareDefense.Models.FirmwareAnalysisSummaryType summaryType, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
quite a few of your breaking changes should be caused by this rename - we need to change it back.
You could do it in `rename-mapping` to change the type name back to `FirmwareAnalysisSummaryName`.
The parameter change here might also be breaking - but let us fix the type name change first, and if the CI does not report this parameter name change issue, we could let it slide.

Reply:
So I have been told this name change was made intentionally to improve the API, therefore I think here we should also apply.